plugins/pencilblue/controllers/actions/forgot_password.js

```diff
@@ -43,7 +43,7 @@ module.exports = function ForgotPasswordControllerModule(pb) {
         var ctx = this.getServiceContext();
         ctx.userService = this.userService;
         ctx.siteService = new SiteService(this.getServiceContext());
-        ctx.emailService = new EmailService(this.getServiceContext());
+        ctx.emailService = new EmailService({site: ctx.site});
```

Comment:
Why change this?  The reason we had it to pull the whole service context was so that if the EmailService expanded to need more parameters we wouldn't have to change code where the service was instantiated.